### PR TITLE
AC_Sprayer: replace AP_InertialNav by AHRS

### PR DESCRIPTION
**Pull request by khancyr**

### CODE REVIEW

**Comment by rmackay9** [comment]
The only thing I'd say is that normally we split the vehicle change from the library change.  It's a single line change in Copter so not sure how strict we want to be on this so I'll leave it up to @OXINARF.

I'm happy for this to go in any time.

**Comment by khancyr** [comment]
Ok I will extract the line

File: libraries/AC_Sprayer/AC_Sprayer.h [hunk 1]
```diff
@@ -19,7 +19,7 @@
 #include <AP_Param/AP_Param.h>
 #include <AP_Math/AP_Math.h>
 #include <SRV_Channel/SRV_Channel.h>
-#include <AP_InertialNav/AP_InertialNav.h>     // Inertial Navigation library
+#include <AP_AHRS/AP_AHRS.h>
```
[the remark this answers is not in the document]
Comment (khancyr) [reply libraries/AC_Sprayer/AC_Sprayer.h]: AP_AHRS include AP_AHRS_NavEKF
But AP_AHRS_NavEKF also include AP_AHRS ... Using AP_AHRS include first allow to configure the ahrs correctly. I didn't found one include of AP_AHRS_NavEKF directly

File: libraries/AC_Sprayer/AC_Sprayer.cpp [hunk 2]
```diff
@@ -107,8 +107,11 @@ AC_Sprayer::update()
     }
 
     // get horizontal velocity
-    const Vector3f &velocity = _inav->get_velocity();
-    float ground_speed = norm(velocity.x,velocity.y);
+    Vector3f velocity;
+    if (_ahrs.get_velocity_NED(velocity)) {
```
[the remark this answers is not in the document]
Comment (khancyr) [reply libraries/AC_Sprayer/AC_Sprayer.cpp]: yep you right ...

**Comment by rmackay9** [comment]
I've merged this after some changes:

- separated the vehicle from library codes in the commits
- change the behaviour when the AHRS can't get a velocity to act as if the velocity is zero.  This is better because it allows testing the sprayer even when the AHRS doesn't have a velocity.

I've left the AHRS.h vs AHRS_NavEKK.h as it was in this PR because it seems like we always include AP_AHRS.h in the code in other places.  I don't know what is correct so I leave that for a wider discussion.

**Pull request closed without merging**